CHANGES.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 1.2.2
+
+* Allow passing base64 encoded content to update and create file actions.
```

Comment:
why not just combine these into 1.2.0?

Reply:
Was deploying/testing in my dev, got a habit of bumping version each time. Sorry! Didn't think that bumping up build/bugfix version makes any difference to anyone.

Reply:
resetting it to 1.2.0